app/assets/stylesheets/styles/confirm-dialog.scss

```diff
@@ -31,14 +31,13 @@ body.confirm-dialog-open {
   }
 
   .confirm-dialog {
-    width: 600px;
-    max-width: 90%;
-    margin: 0 auto;
-    text-align: center;
     position: absolute;
+    width: 90%;
     top: 50%;
-    left: 50%;
-    transform: translate(-50%, -50%);
+    left: 5%;
+    text-align: center;
+    transform: translateY(-50%);
+    -webkit-transform: translateY(-50%);
```

Comment:
we could maybe use the autoprefixer rails gem to do all vendor prefixing for us automatically?